sketch.gemspec

```diff
@@ -3,12 +3,13 @@ $:.push File.expand_path("../lib", __FILE__)
 
 Gem::Specification.new do |s|
   s.name        = "sketch"
-  s.version     = '0.2'
+  s.version     = '0.3'
   s.authors     = ["Brandon Fosdick"]
   s.email       = ["bfoz@bfoz.net"]
   s.homepage    = "http://github.com/bfoz/sketch"
   s.summary     = %q{2D mechanical sketches}
   s.description = %q{Sketches used in the creation of mechanical designs}
+  s.required_ruby_version = "~> 2.6.5"
```

Comment:
To make similar future work easier, how about we make this `~> 2.6`? Changes within the minor version context should be backward compatible. Gems should generally have wide version ranges for supported dependencies and the final app runtime should have the specific, trusted versions.

Reply:
Sounds great, will do.  I'll update the others for this as well 👍